requirements_prod.txt

```diff
@@ -1,5 +1,5 @@
 # PYTHON=3.10
-apache-airflow[amazon,postgres,http]==2.3.4
+apache-airflow[amazon,postgres,http]==2.4.0
```

Comment:
```suggestion
apache-airflow[amazon,postgres,http]==2.4.0b1
```

Reply:
This is a patched version. But I'm not sure if this should apply to amazon, postgres and http packages, too.

Reply:
I believe 2.4.0b1 is the _first version in which the fix was introduced_, but it's not the most recent version. Looking at the commits for both, you can see that [v2.4.0b1](https://github.com/apache/airflow/commits/2.4.0b1) was published before [v2.4.0](https://github.com/apache/airflow/commits/2.4.0) (IIRC semver dictates that `X.Y.Zaaa` is a pre-release version where is `X.Y.Z` is the proper release). So using 2.4.0 should be good!